Assets/Scripts/PlayerControls.cs

```diff
@@ -0,0 +1,168 @@
+﻿using UnityEngine;
+using System.Collections;
+
+[RequireComponent( typeof( ActorPhysics ) )]
+public class PlayerControls : MonoBehaviour
+{
+	PlayerActor _actor;
+	ActorPhysics _actorPhysics;
+	bool _jumpButtonDown = false;
+	bool _glideButtonDown = false;
+	bool _isGrabbing = false;
```

Comment:
This might make more sense as _grabbingInputActive. The current name doesn't seem to fit with _jumpButtonDown and _glideButtonDown.